### PR TITLE
fix cors for staging apps

### PR DESCRIPTION
**Pull request by Enjoy2Live**

### CODE REVIEW

File: config/middlewares.js [hunk 1]
```diff
@@ -1,7 +1,34 @@
...
+];
+
 module.exports = [
   'strapi::errors',
   'strapi::security',
-  'strapi::cors',
+  {
+    name: 'strapi::cors',
+    config: {
+      origin: ['https://devlaunchers.org', 'https://staging.devlaunchers.org', 'http://localhost:3000', ...DevLaunchersSubdomains],
```
Comment (chungthuang) [inline config/middlewares.js:26]: Nit: Technically https://staging.devlaunchers.org is also a subdomain.

Comment (Enjoy2Live) [reply config/middlewares.js:26]: added while you're writing this :laughing:

Comment (Enjoy2Live) [reply config/middlewares.js:26]: @chungthuang are you going to approve this now?

**Review by chungthuang — APPROVED** [review]

**Pull request merged**